src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+  ],
+  "page": 1,
+  "total": 1,
+  "pageSize": 5
+}
+```
+
+#### Enrollments single object endpoint `GET /api/tracker/enrollments/{uid}`
+
+Purpose of this endpoint is to retrieve one enrollment given its uid.
+
+##### Request syntax
+
+`GET /api/tracker/enrollment/{uid}?fields={fields}`
+
+|Request parameter|Type|Allowed values|Description|
+|---|---|---|---|
+|`uid`|`String`|`uid`|Return the Enrollment with specified `uid`|
+|`fields`|`String`| **Not implemented yet**|Include specified sub-objects in the response| 
+
+##### Example requests
+
+A query for a Enrollment:
+
+    GET /api/tracker/enrollments/iKaBMOyq7QQ
+
+##### Response format
+
+```json
+{
+  "enrollment": "iKaBMOyq7QQ",
+  "createdAt": "2017-03-28T12:28:19.812",
+  "createdAtClient": "2016-03-28T12:28:19.812",
+  "updatedAt": "2017-03-28T12:28:19.817",
+  "trackedEntity": "PpqV8ytvW5i",
+  "trackedEntityType": "nEenWmSyUEp",
+  "program": "ur1Edk5Oe2n",
+  "status": "ACTIVE",
+  "orgUnit": "NnQpISrLYWZ",
+  "orgUnitName": "Govt. Hosp. Bonthe",
+  "enrolledAt": "2020-10-23T12:28:19.805",
+  "occurredAt": "2020-10-07T12:28:19.805",
+  "followUp": false,
+  "deleted": false,
+  "events": [],
+  "relationships": [],
+  "attributes": [],
+  "notes": []
+}
+```
+
+### Relationships (`GET /api/tracker/relationships`)
+
+Relationships are links between two entities in tracker.
+These entities can be tracked entity instances, enrollments and events.
+
+Purpose of this endpoint is to retrieve Relationships between objects.
+
+Unlike other tracked objects endpoints, Relationship only expose one endpoint:
+
+- `GET /api/tracker/relationships?[tei={teiUid}|enrollment={enrollmentUid}|event={eventUid}]&fields=[fields]`
+
+#### Request parameters
+|Request parameter|Type|Allowed values|Description|
+|---|---|---|---|
+|`tei`|`String`|`uid`| Identifier of a Tracked Entity Instance|
+|`enrollment`|`String`|`uid`| Identifier of an Enrollment |
+|`event`|`String`|`uid`| Identifier of and Event|
+|`fields`|`String`| | **Not implemented yet:** Only includes specified properties in the response| 
+
+The following rules apply to the query parameters.
+
+- only one parameter among `tei`,`enrollment`,`event` can be passed
+
+> **NOTE:**
+>
+> using tei, enrollment or event params, will return any relationship where the tei\enrollment or
+> event is part of the relationship (either from or to). As long as user has access that is.
+> 
+
+#### Example response
+
+```json
+[
+  {
+    "relationshipType": "dDrh5UyCyvQ",
+    "relationshipName": "Mother-Child",
+    "relationship": "t0HIBrc65Rm",
+    "bidirectional": false,
+    "from": {
+      "trackedEntity": {
+        "trackedEntity": "vOxUH373fy5"
+      }
+    },
+    "to": {
+      "trackedEntity": {
+        "trackedEntity": "pybd813kIWx"
+      }
+    },
+    "created": "2019-04-26T09:30:56.267",
+    "lastUpdated": "2019-04-26T09:30:56.267"
+  },
+  ...,
+]
+```
+
+## Tracker Access Control
+
+<!--DHIS2-SECTION-ID:webapi_nti_access_control-->
```

Comment:
empty section ?

Reply:
added an "introduction" to the following sections here